docs/core/tools/dotnet-new.md

```diff
@@ -132,8 +132,9 @@ Filters templates based on available types. Predefined values are "project", "it
 Uninstalls a source or template pack at the `PATH` or `NUGET_ID` provided.
 
 > [!NOTE]
-> To uninstall a template using a `PATH`, you need to fully qualify the path. For example, *C:/Users/\<USER>/Documents/Templates/GarciaSoftware.ConsoleTemplate.CSharp* will work, but *./GarciaSoftware.ConsoleTemplate.CSharp* from the containing folder will not.
-> Additionally, do not include a final terminating directory slash on your template path.
+> To uninstall a template using a source `PATH`, you need to fully qualify the path. For example, *C:/Users/\<USER>/Documents/Templates/GarciaSoftware.ConsoleTemplate.CSharp* will work, but *./GarciaSoftware.ConsoleTemplate.CSharp* from the containing folder will not. Additionally, do not include a final terminating directory slash on your template path.
+> 
+> Even if you installed a template using a `PATH` to a local NuGet *.nupkg* package, you will still uninstall it from the package's `NUGET_ID` rather than the source path of the *.nupkg* file.
```

Comment:
That's a valid question. It is definitely confusing to me, hence the two different PRs I have submitted to just this one option.

If I understand it correctly, you can install from a NuGet package or directly from a "source" folder on your local machine set up with a `.template.config` folder.

For a NuGet package, you can install from nuget.org by just providing a NuGet package ID (`NUGET_ID`), which gets you a `*.nupkg` archive to use, or you can install from a `*.nupkg` archive on your local machine (via a `PATH` value, but installing from an archive). Regardless of how you get the nupkg archive, you uninstall the same template by it's NuGet package ID., even if you installed it via a `PATH` initially. **This is what this PR's addition was intended to clarify.**

For a "source" folder, you install it from a local path (`PATH`). While you can install from a relative path to your current working directory, you can only uninstall from an absolute path. That is what the prior note addition was intended to clarify.

That is the understanding I have of things from poking at them and making all sorts of mistakes. It would be nice to have a `dotnet new --list` option that could show the appropriate parameter needed to uninstall every item in the list.

Reply:
Thanks for the clarification @patridge. Also adding @livarcocc to see who would be appropriate to help with this question and see if a feature request should be filed in the product repo too. 

If uninstalling with PATH only works for a certain scenario and not for *.nupkg files, then we can change the first sentence in the note to call out that. I didn't use this feature enough myself to know unfortunately.

Reply:
Running `dotnet new -u` without any parameters gives the list of the valid strings to include as the parameter for uninstalling a specific template pack.

Reply:
The output from `dotnet new -u` will indicate the exact strings needed as parameters for uninstalling. They'll include / exclude the path / Id as needed. Might be easiest to just tell the user that, instead of the rules for when each format is needed.